### PR TITLE
close #301 docs(experimenter): aug Newsletter

### LINKED ISSUE #301: August "What's Newsletter"
Create a "What's Newsletter" for August 2022

### PR DESCRIPTION
**Pull request by mikewilli**
What's Newsletter for August 2022

Resolved issues list:
https://mozilla-hub.atlassian.net/jira/software/c/projects/EXP/issues/?jql=project%20%3D%20%22EXP%22%20AND%20status%20%3D%20%22Done%22%20AND%20resolved%20%3E%3D%20%222022-08-01%22%20AND%20resolved%20%3C%3D%20%222022-09-01%22%20ORDER%20BY%20created%20DESC

### CODE REVIEW

File: docs/whats-news/2022/2022-08.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# What's New in Nimbus August 2022
+
+## Welcome
+
+The Nimbus team will have a **demo booth at #moz-aloha** All-Hands at the end of September! Come find us, chat about experimentation, and maybe even grab a sticker!
+
+## Exciting Changes
+
+### Mobile First Run
```
Comment (yashika-khurana) [inline docs/whats-news/2022/2022-08.md:15]: I think we should add here little info what does mobile-first run feature mean

File: docs/whats-news/2022/2022-08.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- This was affecting more than 100,000 clients and over 1.5M events/requests.
+
+### Cancel Review
+
+If you launch, end, or end enrollment for an experiment it automatically goes to Review. Now, there is an option
+to cancel that change and go back to an idle state where changes can be made.
+
+### UI/UX Updates
+
+Various user experience updates to streamline workflows and provide more consistency throughout the tool, like:
```
Comment (yashika-khurana) [inline docs/whats-news/2022/2022-08.md:27]: Also in terms of UI improvements, you could add `You can see all dates with the year now`- something like that

**Review by yashika-khurana — CHANGES_REQUESTED** [review]
Thank you @mikewilli for handling this, a few suggestions- we can add a section for `Rollouts`

File: docs/whats-news/2022/2022-08.md (hunk 1 above)
Comment (mikewilli) [reply docs/whats-news/2022/2022-08.md:15]: Sounds like a good idea! I added a sentence about it to the latest commit, let me know if you have any suggestions to improve wording/clarity.

File: docs/whats-news/2022/2022-08.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+The Nimbus team will have a **demo booth at #moz-aloha** All-Hands at the end of September! Come find us, chat about experimentation, and maybe even grab a sticker!
+
+## Exciting Changes
+
+### Mobile First Run
+
+Mobile First Run refers to the work done in Nimbus to allow experiments during an user's first run of the Firefox mobile app.
+- Instrument Nimbus SDK in telemetry in order to provide experimental treatments during the onboarding experience.
+  - This was affecting more than 100,000 clients and over 1.5M events/requests.
```
Comment (yashika-khurana) [inline docs/whats-news/2022/2022-08.md:19]: do you think spacing gonna impact UI? 🤔

File: docs/whats-news/2022/2022-08.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- Performance improvements
+- All displayed dates now include the year
+
+### Improved Testing and Validation
+
+- Better integration testing
+- Better validation of configurations
+
+---
+
```
Comment (yashika-khurana) [inline docs/whats-news/2022/2022-08.md:40]: Rollouts section is still missing, let me know if you need help with that

**Review by yashika-khurana — CHANGES_REQUESTED** [review]
thanks @mike, Rollouts section is still missing :(

File: docs/whats-news/2022/2022-08.md (hunk 3 above)
Comment (yashika-khurana) [reply docs/whats-news/2022/2022-08.md:19]: yeah I was not even sure, but I think this will not impact

File: docs/whats-news/2022/2022-08.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+### Improved Rollout Support
+
+A Rollout is an experiment where instead of configuring control and treatment branches, a feature is slowly rolled out to a population of clients. Experimenter now has improved support for Rollouts.
+- A client can:
+  - enroll in a rollout, receive feature values, send telemetry
+  - explicitly unenroll from the client UI
+  - be automatically unenrolled by unpublishing the record from RS
+  - enroll in a rollout for a feature, and then ALSO enroll in an experiment for the same feature, receive the experiment feature configuration while enrolled in the experiment, then be unenrolled from the experiment and fall back to the rollout feature configuration
+- User Interface
+  - Now just check a box to make your experiment a rollout (for supported clients).
```
Comment (yashika-khurana) [inline docs/whats-news/2022/2022-08.md:30]: interesting, here are the list of supported clients 
```
    ROLLOUT_SUPPORT_VERSION = {
        Application.DESKTOP: Version.FIREFOX_105,
        Application.FENIX: Version.FIREFOX_105,
        Application.FOCUS_ANDROID: Version.FIREFOX_105,
        Application.IOS: Version.FIREFOX_105,
        Application.FOCUS_IOS: Version.FIREFOX_105,
    }
```
In short for every application we are supporting from 105 version

**Review by yashika-khurana — CHANGES_REQUESTED** [review]
Just a minor suggestion to improve rollouts section :)

File: docs/whats-news/2022/2022-08.md (hunk 5 above)
Comment (mikewilli) [reply docs/whats-news/2022/2022-08.md:30]: Great! There was a note in one of the PRs about client support so I put that note in. I'll just remove that note, thanks!

**Pull request merged**